pandas/core/indexes/base.py

```diff
@@ -200,7 +200,8 @@ def __new__(cls, data=None, dtype=None, copy=False, name=None,
         # interval
         if is_interval_dtype(data) or is_interval_dtype(dtype):
             from .interval import IntervalIndex
-            return IntervalIndex(data, dtype=dtype, name=name, copy=copy)
+            return IntervalIndex(data, dtype=dtype, name=name, copy=copy,
+                                 **kwargs)
```

Comment:
do we actually need kwargs to be passed or is that for consistentcy with others?

Reply:
It's needed to pass `closed`.

Reply:
can you pass that directly?

Reply:
Done by doing `closed = kwargs.get('closed', None)`, then passing that into the constructor.  Not sure if you meant doing it that way, or by adding `closed=None` to the `Index` constructor.